Recast/AppDelegate.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AppDelegate.swift
+//  Recast
+//
+//  Created by Drew Dunne on 9/12/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        
+        window = UIWindow()
+        
+        let seriesViewControllerTest = SeriesViewController(nibName: nil, bundle: nil)
+        let nc = UINavigationController(rootViewController: seriesViewControllerTest)
```

Comment:
can you name this variable better? even though it's gonna be replaced later just for style conventions